src/jwt/parseJWTClient.ts

```diff
@@ -74,9 +74,6 @@ export async function parseJWT(token: string, publicKey: string, timeOffset: num
         const payload = readJWTPayload(token)
 
         if (!(
-            typeof payload.usrnm === 'string' &&
```

Comment:
I think this file is wrongly placed. First I think the function should be called parseJWTClient and this parse is specific to omegaId so why is it placed with the general jwt stuff. I think this should be split: one part is. in jwt libary it verifies and parses jwt on client. One part decodes and then uses this function - this should be placed with omegaId functionality.